certification/engine/engine.go

```diff
@@ -88,6 +88,21 @@ var nameToChecksMap = map[string]certification.Check{
 	validateOperatorBundle.Name():         validateOperatorBundle,
 }
 
+var containerPolicyChecks = map[string]certification.Check{
+	runAsNonRootCheck.Name():              runAsNonRootCheck,
+	underLayerMaxCheck.Name():             underLayerMaxCheck,
+	hasRequiredLabelCheck.Name():          hasRequiredLabelCheck,
+	basedOnUbiCheck.Name():                basedOnUbiCheck,
+	hasLicenseCheck.Name():                hasLicenseCheck,
+	hasMinimalVulnerabilitiesCheck.Name(): hasMinimalVulnerabilitiesCheck,
+	hasUniqueTagCheck.Name():              hasUniqueTagCheck,
+	hasNoProhibitedCheck.Name():           hasNoProhibitedCheck,
+}
+
+var operatorPolicyChecks = map[string]certification.Check{
+	validateOperatorBundle.Name(): validateOperatorBundle,
+}
+
 func AllChecks() []string {
```

Comment:
I'm assuming that functions like this one remain because the cobra command split hasn't happened yet. Do you intend on removing this code when the split happens?

If not, I'll assume that the `nameToChecksMap` (higher up in this file) _will_ go away regardless of whether we get rid of this function. With that in mind, do we want to go ahead and have you change this logic to sum return a sum of the values from `containerPolicyChecks` ard `operatorPolicyChecks` instead?

Reply:
Yeah.  I was trying to make small changes but agreed.  In a future PR I was thinking about just creating a `func getChecks(policy string)` Where we could pass in the policy and it would just return the checks.